epsagon/handler.py

```diff
@@ -0,0 +1,29 @@
+"""
+Epsagon generic wrapper used only in Lambda environments.
+"""
+
+from .utils import init, import_original_module
+from .wrappers import lambda_wrapper
+
+init()
+ORIGINAL_MODULE = import_original_module()
+
+
+@lambda_wrapper
+def wrapper(event, context):
```

Comment:
maybe use the lambda_wrapper only on the user's handler so we won't be in the trace? not sure it's a good idea but think about it.

Reply:
Done.

Reply:
this is not what i meant. I meant the code should look like this:
```
wrapper = lambda_wrapper(init_module())
```
and then just set wrapper as the entry point (maybe call it EPSAGON_WRAPPER)